apps/taxes/src/modules/avatax/order-confirmed/avatax-order-confirmed-payload-lines-transformer.ts

```diff
@@ -1,26 +1,37 @@
 import { LineItemModel } from "avatax/lib/models/LineItemModel";
-import { OrderConfirmedSubscriptionFragment } from "../../../../generated/graphql";
+import {
+  OrderConfirmedSubscriptionFragment,
+  OrderLineFragment,
+} from "../../../../generated/graphql";
 import { numbers } from "../../taxes/numbers";
 import { AvataxConfig } from "../avatax-connection-schema";
 import { AvataxTaxCodeMatches } from "../tax-code/avatax-tax-code-match-repository";
-import { SHIPPING_ITEM_CODE } from "./avatax-order-confirmed-payload-transformer";
 import { AvataxOrderConfirmedTaxCodeMatcher } from "./avatax-order-confirmed-tax-code-matcher";
 
+import { SHIPPING_ITEM_CODE } from "../calculate-taxes/avatax-calculate-taxes-adapter";
+
+/**
+ * @description Makes sure we use the same line number for creating & refunding transaction. AvaTax requires that the line number is <= 50 characters.
+ */
+export function resolveAvataxTransactionLineNumber(line: OrderLineFragment) {
+  return line.id.slice(-50);
+}
+
```

Comment:
suggestion: Why not use `Order.token` instead of shortening the ID?

Here's the comparison in length between token (UUID from Saleor database) and ID (base64 of Order:<UUID>)

```
❯ echo "951feaa9-d613-4e0c-bd6a-373134ec3dc1" | wc -c                       
      37
❯ echo "T3JkZXI6OTUxZmVhYTktZDYxMy00ZTBjLWJkNmEtMzczMTM0ZWMzZGMx" | wc -c                                    
      57
```

In the schema it's marked as deprecated, but since we still keep on finding use-cases for shorter IDs we might leave it (conversation: https://saleorcommerce.slack.com/archives/C0352SN0W3U/p1706698862183379)